CHANGELOG.md

```diff
@@ -12,11 +12,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `quint verify` has the option `--apalache-version` to pull a custom version (#1521)
 
 ### Changed
+
+- Updated grammar rule to allow an optional trailing comma in parameter lists:
```

Comment:
It's good to add a reference to an issue/PR for whoever needs more context and details.
```suggestion
- Updated grammar rule to allow an optional trailing comma in parameter lists (#1510):
```